src/Stripe.Tests.XUnit/orders/_fixture.cs

```diff
@@ -27,7 +27,7 @@ public orders_fixture()
                     {
                         Amount = 1,
                         Description = "some random items in a furry sack",
-                        Parent = "sku_stripe_net_test", // i hardcoded this because skus isn't written yet. created from the dashboard
+                        Parent = "sku_B3QG08Tr9vICLA", // i hardcoded this because skus isn't written yet. created from the dashboard
```

Comment:
Please remove this mod, as the `sku_stripe_net_test` actually exists on the test account.